open-machine-learning-jupyter-book/assignments/data-science/Displaying-airport-data.ipynb

```diff
@@ -0,0 +1,144 @@
+{
```

Comment:
<p>You can use the content from here as a description? <a href="https://github.com/microsoft/Data-Science-For-Beginners/blob/main/2-Working-With-Data/05-relational-databases/assignment.md#database-schema" target="_blank">https://github.com/microsoft/Data-Science-For-Beginners/blob/main/2-Working-With-Data/05-relational-databases/assignment.md#database-schema</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/263/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>